gulpfile.js

```diff
@@ -148,7 +148,9 @@ gulp.task('installFixtures', function() {
     }, 1 * 1000);
     shell.cd('test/fixtures');
 
-    execAsync('npm install --quiet', {cwd: '../fixtures'}).then(() => {
+    execAsync('type yarn &> /dev/null | yarn install || npm install --quiet', {
```

Comment:
I'm not on windows and don't have access to it atm.
I found this: https://superuser.com/questions/175466/determine-if-command-is-recognized-in-a-batch-file
Hence we could use: `WHERE yarn >nul 2>nul; IF %ERRORLEVEL% EQU 0 yarn install ELSE npm install --quiet;`
Can you or sb. confirm that this works please?